### PR TITLE
CTCP-1479: update pre task list content

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -838,7 +840,7 @@ routeDetails.locationOfGoods.postalCode.postalCode = Postal code
 routeDetails.locationOfGoods.postalCode.country = Country
 routeDetails.locationOfGoods.postalCode.checkYourAnswersLabel = What is the address for the location of goods?
 routeDetails.locationOfGoods.postalCode.change.hidden = the address for the location of goods
-routeDetails.locationOfGoods.postalCode.error.streetNumber.invalid = The street number must only include numbers 0 to 9 and letters a to z
+routeDetails.locationOfGoods.postalCode.error.streetNumber.invalid = The street number must only include numbers 0 to 9 and lettersa a to z
```
Comment (jackheslop96) [inline conf/messages.en:843]: Typo?

Comment (kenneth-rayner) [reply conf/messages.en]: done cheers

**Comment by kenneth-rayner** [comment]
rebuild

**Comment by kenneth-rayner** [comment]
rebuild

**Comment by kenneth-rayner** [comment]
rebuild

**Review by jackheslop96 — APPROVED** [review]

**Pull request merged**